Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```
@@ -161,7 +162,7 @@
   name: Kammerer
   parent: [BaseWeaponShotgun, BaseGunWieldable, BaseRestrictedContraband]
   id: WeaponShotgunKammerer
-  description: When an old Remington design meets modern materials, this is the result. A favourite weapon of militia forces throughout many worlds. Uses .50 shotgun shells.
+  description: A popular shotgun found on many worlds. While its technology is wildly outdated by modern standards, the utility and simplicity of a pump shotgun are still valuable in many roles. Its true strength comes in the versatility of shell types it can fire, from pellets to slugs and even flares.
```

```suggestion
  description: A popular shotgun found on many worlds. While its technology is wildly outdated by modern standards, the utility and simplicity of a pump shotgun are still valuable in many roles.
```
any shotgun can do this

---

changed to 
"A popular shotgun found on many worlds. While its technology is wildly outdated by modern standards, the utility and simplicity of its design have kept it in service to this day."
hows that?